internetarchive/item.py

```diff
@@ -51,6 +51,16 @@
 from internetarchive.iarequest import MetadataRequest, S3Request
 from internetarchive.utils import get_s3_xml_text, get_file_size, is_dir
 
+# Progress bars using https://github.com/tqdm/tqdm
+# Import tqdm without enforcing it as a dependency
+try:
+    from tqdm import tqdm
+except ImportError:
+    def tqdm(*args, **kwargs):
+        if args:
+            return args[0]
+        return kwargs.get('iterable', None)
+
```

Comment:
it might be worth doing that magic only once, in one location, otherwise bitrot might break it.... maybe we could import from utils here instead?